clients/cobol-lsp-vscode-extension/package.json

```diff
@@ -190,6 +174,20 @@
                     "description": "Default list of USS paths to search for IDMS copybooks\nZowe Explorer version 1.15.0 or higher is required to download copybooks from the mainframe",
                     "uniqueItems": true
                 },
+                "cobol-lsp.cpy-manager.copybook-extensions": {
+                    "type": "array",
+                    "items": {
+                        "type": "string"
+                    },
+                    "default": [
+                        ".CPY",
+                        ".COPY",
+                        ".cpy",
+                        ".copy"
+                    ],
+                    "description": "List of copybook extensions",
```

Comment:
@zeibura, Could you please verify the description. This should be a list expected by LSP server extensions of files with copybooks.

Reply:
Looks good to me, this will require a readme update I suppose? Is it a new setting?

Reply:
Yes, that's a new one

Reply:
OK I will open a PR for this